### DISCUSSION TITLE
Keypass on Sourceforge vs KeypassXC

### DISCUSSION THREAD

**Discussion by the-moog** [keepassxreboot/keepassxc#9492] [Q&A]
I'm confused (or was??) by KeyPass releases.

I am new to this project, in looking for a non Google/Samsung password manager that I can self host as I don't trust cloud service providers. Too many have screwed up with zero accountability or liability.

I use Firefox right now but that does not work well with other apps and has a habit of storing the wrong data. - Though I trust Mozilla more than the others, at least I know the data leaves my network encrypted.

Anyway, correct me if I a wrong, I think I've worked it out (XC is in a better language), but perhaps (and perhaps a bug) the README.md first couple lines should clear up that KeyPass and KeyPassXC though they share a name are far from the same thing?

At first I was confused as KeyPass is at version 2.53 on Sourceforge and KeyPassXC is at version 2.7.2, so it looks like KeyPass is way newer and KeyPassXC is some old project - but then looking at the commits and bugs I can see that is not true.  Fairly sure I know what I am doing when it comes to software, so if it can confuse me I am sure others are confused too?

Not that it's my problem, but a name that indicates that KeyPassXC is 'Next Gen' might have been better.  - Change it to KeyPassNG perhaps?

**Comment by michaelk83** [comment]
KeyPassXC isn't next gen compared to KeyPass, they're separate projects. They use the same database format and have similar features, so you can pretty much interchange them based on your needs and preferences. There are also other compatible variants such as for mobile.

KeePassXC vs KeePass vs KeePassX is explained [in the FAQ](https://keepassxc.org/docs/). I agree that the readme could at least mention that FAQ, but it does already link to the website where the FAQ is found.

**Reply by the-moog** [reply]
Yep, got that. Found the FAQ once I stumbled across this repo.  Just commenting that from my experience of writing a *lot* of technical documentation, that it's best to write from the perspective of the reader and assume limited prior knowledge. Get confusion out of the way on page one rather than in the appendix.  I am sure you know how engineers love to fully RTM rather than risk ploughing in and breaking stuff...... :o)
